src/LeftNav/right-arrow.svg

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
> webpack builds it into the component.

100% positive on that? we build to dist with plain babel, not webpack, so i think clients would have to manually include it in their webpack setup